c2corg_ui/static/js/auth/authservice.js

```diff
@@ -260,6 +260,16 @@ app.Authentication.prototype.addAuthorizationToHeaders = function(url,
 
 
 /**
+ * @export
+ */
+app.Authentication.prototype.getUserToken = function() {
+  if (this.userData) {
+    return this.userData.token;
+  }
```

Comment:
The function should probably return something when ``this.userData`` is not set.